tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestCategoryAttributeStruct.cs

```diff
@@ -6,6 +6,8 @@
 using System.Collections.Generic;
 using Datadog.Trace.DuckTyping;
 
+#pragma warning disable CS0649
```

Comment:
I wonder, why haven't we seen these errors on other structs? 🤔 

Also, we should probably `#nullable enable` where possible in these 

```suggestion
#pragma warning disable CS0649 // Field 'TestCategories' is never assigned to
```